packages/app/app/components/Dashboard/BestNewMusicTab/BestNewMusicContent/index.tsx

```diff
@@ -30,7 +36,7 @@ const BestNewMusicContent = ({
           </div>
           <div
             className={styles.title}
-            onClick={() => albumInfoSearchByName(item.title + ' ' + item.artist, history)}
+            onClick={() => albumInfoSearchByName(item.title + ' ' + item.artist, item.artist,  history)}
```

Comment:
Not sure that this line is correct 🤔 the signature for albumInfoSearchByName (in search.ts) is (title, artist, history) - should item.title be concatenated with item.artist (as the first argument) here?